var/spack/repos/builtin/packages/r-smoof/package.py

```diff
@@ -0,0 +1,51 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RSmoof(RPackage):
+    """Provides generators for a high number of both single- and
+       multi- objective test functions which are frequently used for the
+       benchmarking of (numerical) optimization algorithms. Moreover, it offers
+       a set of convenient functions to generate, plot and work with objective
+       functions."""
+
+    homepage = "http://github.com/jakobbossek/smoof"
+    url      = "https://cran.r-project.org/src/contrib/Archive/smoof/smoof_1.5.tar.gz"
```

Comment:
Can you add the latest version:
https://cran.r-project.org/src/contrib/smoof_1.5.1.tar.gz

This is causing the concretization error you are seeing.